tron/core/jobrun.py

```diff
@@ -179,6 +192,7 @@ def seconds_until_run_time(self):
 
     def start(self):
         """Start this JobRun as a scheduled run (not a manual run)."""
```

Comment:
What method does a manual run invoke? Will it get logged?

Reply:
Yes, manual runs invoke this method too.

Reply:
Can you correct the docstring then?